tests/modules/programs/yazi/settings.nix

```
@@ -70,7 +70,10 @@
       };
     };
     initLua = ./init.lua;
-    plugins = { "test.yazi" = ./plugin; };
+    plugins = {
+      "test.yazi" = ./plugin;
+      "anotherTest" = ./plugin;
```

```suggestion
      "test" = ./plugin;
      "another-test" = ./plugin;
```

Yazi's plugin/package names should only include kebab-case characters, so it should be `another-test` instead of `anotherTest` here.

> Each plugin is a directory with a [kebab-case](https://developer.mozilla.org/en-US/docs/Glossary/Kebab_case) name, ending in .yazi